src/app/src/main/kotlin/org/imperial/mrc/hint/security/SecurityConfig.kt

```diff
@@ -7,9 +7,11 @@ import org.springframework.web.servlet.config.annotation.InterceptorRegistry
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
 
 @Configuration
-open class SecurityConfig(val config: Config): WebMvcConfigurer {
+class SecurityConfig(val config: Config): WebMvcConfigurer {
+
     override fun addInterceptors(registry: InterceptorRegistry) {
         registry.addInterceptor(SecurityInterceptor(config, "FormClient"))
-                .addPathPatterns("/")
+                .addPathPatterns("/**")
+                .excludePathPatterns("/login", "/login/", "/password/**", "/callback", "/callback/")
```

Comment:
For the life of me could not get automatic trailing slash matching to work so gave up and specified with and without trailing slashes explicitly.